package.json

```diff
@@ -20,7 +20,7 @@
     "node.extend": "^1.1.7"
   },
   "devDependencies": {
-    "@dvsa/mes-journal-schema": "^1.0.8",
+    "@dvsa/mes-journal-schema": "1.0.10",
```

Comment:
Should be using version 1.1.0 on merges into develop and 1.0.10 on cherry picks into release 2

Reply:
I'm assuming that's what your merge conflicts are saying ;)

Reply:
Have changed to use 1.1.0